covalent_slurm_plugin/slurm.py

```diff
@@ -70,18 +70,18 @@ class SlurmExecutor(AsyncBaseExecutor):
     Args:
         username: Username used to authenticate over SSH.
         address: Remote address or hostname of the Slurm login node.
-        ssh_key_file: Private RSA key used to authenticate over SSH.
-        cert_file: Certificate file used to authenticate over SSH, if required.
+        ssh_key_file: Private RSA key used to authenticate over SSH (usually at ~/.ssh/id_rsa)
+        cert_file: Certificate file used to authenticate over SSH, if required (usually has extension .pub).
         remote_workdir: Working directory on the remote cluster.
         slurm_path: Path to the slurm commands if they are not found automatically.
-        conda_env: Name of conda environment on which to run the function.
+        conda_env: Name of conda environment on which to run the function. Use "" for the base environment or False for no conda.
```

Comment:
I think it would be more clear if `conda_env="base"` specifies usage of the base environment.  We can check for this special case and transform it to `""` in the constructor.  Any of `False`, `None`, or `""` should be equivalent in not using Conda.  This should simplify some of the conditional logic.

Reply:
I was pretty torn on it so glad to hear you've picked a side! Done deal.

Reply:
I don't think there is a clean way to have it such that `None` would return no Conda because `None` (or not setting anything at all) has to pull up the default config variables. But right now, False and "" both avoid the use of Conda, which I think is suitable. The type for the kwarg is `str` anyway.